fe/fe-core/src/main/java/com/starrocks/credential/CloudConfigurationFactory.java

```diff
@@ -44,6 +46,12 @@ public static CloudConfiguration buildCloudConfigurationForStorage(Map<String, S
             return cloudConfiguration;
         }
 
+        factory = new HDFSCloudConfigurationFactory(properties);
```

Comment:
do we add tests in `CloudConfigurationFactory` to cover the build of new cloud configuration.

Reply:
Tests have been added in `S3StorageParamsTest` and `HDFSStorageparamsTest`.